tutorials/xr/basic_xr_locomotion.rst

```diff
@@ -3,5 +3,65 @@
+
+.. image:: img/godot_xr_tools_floor.webp
+
+Direct movement
+---------------
+
+We're going to start adding some basic direct movement to our setup. This allows the user to move through the virtual world using joystick input.
+
+.. note::
+  It is important to note that moving through the virtual world while the player is standing still in the real world can be nausea inducing especially for players who are new to VR. The default settings on our movement functions are fairly conservative. We advise you to stick to these defaults but offer features in game to enable less comfortable settings for more experienced users who are used to playing VR games.
+
+We want to enable this on the right hand controller and we do this by adding a subscene to the right hand :ref:`XRController3D <class_xrcontroller3d>` node. Select ``addons/godot-xr-tools/functions/movement_direct.tscn`` as the scene to add.
+
+This function adds forward and backwards movement to the player by using the joystick on the right hand controller. It has an option to also add left/right strafe but by default this is disabled.
+
+Instead we are going to add the ability for the player to also turn with this joystick so we will add another subscene to our controller node, select ``addons/godot-xr-tools/functions/movement_turn.tscn`` for this.
+
+The turn system by default uses a snap turn approach. This means that turning happens in steps. This may seem jarring however it is a tried and tested method of combating motion sickness. You can easily switch to a mode that offers smooth turning by changing the ``mode`` property on the turn node.
+
+If you run your game at this point in time you will find that you can move through the world freely using the right hand joystick.
+
+Teleport
+--------
+
+An alternative to direct movement that some users find more pleasant is the ability to teleport to another location within your game world. Godot XR Tools supports this through the teleport function and we will be adding this to our left hand controller.
+
+Add a new child scene to your left hand :ref:`XRController3D <class_xrcontroller3d>` node by selecting the ``addons/godot-xr-tools/functions/function_teleport.tscn`` scene.
+
+With this scene added the player will be able to teleport around the world by pressing the trigger on the left hand controller, pointing where they want to go, and then releasing the trigger.
+The player can also adjust the orientation by using the left hand controllers joystick.
+
+If you've followed all instructions correctly your scene should now look something like this:
+
+.. image:: img/godot_xr_tools_basic_movement.webp
+
+More advanced movement features
+-------------------------------
+
+Godot XR Tools adds many more movement features such as gliding, a grapple hook implementation (go Spidey!), a jetpack, climbing mechanics, etc.
```

Comment:
```suggestion
Godot XR Tools adds many more movement features such as gliding, a grapple hook implementation, a jetpack, climbing mechanics, etc.
```
Given how extremely protective Disney can be of their IP we should probably leave this out. It also looks a little unprofessional.

Reply:
Lol, yeah you're right :)